<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>mainwindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow {
    background-color: green;
}</string>
  </property>
  <widget class="QWidget" name="menuContainer" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>600</height>
    </rect>
   </property>
   <widget class="QPushButton" name="guestButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>314</width>
      <height>53</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Custom button style */
QPushButton {
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
    font-size: 30px;
}

QPushButton:hover {
    color: #53D09E
}</string>
    </property>
    <property name="text">
     <string>Войти как гость</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="enterButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>310</y>
      <width>314</width>
      <height>53</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Custom button style */
QPushButton {
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
    font-size: 30px;
}

QPushButton:hover {
    color: #53D09E
}</string>
    </property>
    <property name="text">
     <string>Войти</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="logoLabel">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>181</width>
      <height>181</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Qlabel {
    border: none;
    background: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../bsuir/ПНАЯВУ/logo2.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="regButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>370</y>
      <width>314</width>
      <height>53</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>-1</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">/* Custom button style */
QPushButton {
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
    font-size: 30px;
}

QPushButton:hover {
    color: #53D09E
}</string>
    </property>
    <property name="text">
     <string>Зарегестрироваться</string>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="default">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="introText">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>170</y>
      <width>621</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Yu Gothic UI Light</family>
      <pointsize>30</pointsize>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
    border: none;
    background: none;
    color: white;
    font-family: 'Yu Gothic UI Light', sans-serif;
}</string>
    </property>
    <property name="text">
     <string>Добро пожаловать на ИОН</string>
    </property>
   </widget>
   <widget class="QLabel" name="background">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>851</width>
      <height>671</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">Qlabel {
    border: none;
    background: none;
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Plain</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>resources/bg1.jpg</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
    </property>
    <property name="wordWrap">
     <bool>false</bool>
    </property>
    <property name="openExternalLinks">
     <bool>false</bool>
    </property>
   </widget>
   <zorder>background</zorder>
   <zorder>guestButton</zorder>
   <zorder>enterButton</zorder>
   <zorder>logoLabel</zorder>
   <zorder>regButton</zorder>
   <zorder>introText</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>enterButton</sender>
   <signal>clicked()</signal>
   <receiver>enterButton</receiver>
   <slot>showMenu()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>288</x>
     <y>333</y>
    </hint>
    <hint type="destinationlabel">
     <x>484</x>
     <y>336</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
